decidim-initiatives/app/types/decidim/initiatives/initiative_type.rb

```diff
@@ -26,9 +25,9 @@ module Initiatives
       field :signatureType, types.String, "Signature type of the initiative", property: :signature_type
       field :signatureStartDate, !Decidim::Core::DateType, "The signature start date", property: :signature_start_date
       field :signatureEndDate, !Decidim::Core::DateType, "The signature end date", property: :signature_end_date
-      field :offlineVotes, types.Int, "The number of offline votes in this initiative", property: :offline_votes
-      field :initiativeVotesCount, types.Int, "The number of votes in this initiative", property: :initiative_votes_count
-      field :initiativeSupportsCount, types.Int, "The number of supports in this initiative", property: :initiative_supports_count
+      field :offlineVotes, types.Int, "The number of offline votes in this initiative", property: :offline_votes_count
+      field :initiativeVotesCount, types.Int, "The number of votes in this initiative", property: :online_votes_count
+      field :initiativeSupportsCount, types.Int, "The number of supports in this initiative", property: :supports_count
```

Comment:
Is this supposed to be `offlineVotes + initiativeVotesCount`? If so, could you update the field descrpition? 😄

Reply:
Looking at this, it may be as well deleted ... 

![image](https://user-images.githubusercontent.com/105683/102513902-7bb92200-4094-11eb-82cd-c951e9deb5ee.png)

`online_votes: { "total" => 3 }` is the value returned by `online_votes_count` 
`offline_votes: { "total" => 4 }` is the value returned by `offline_votes_count`